api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -389,19 +390,32 @@ private User initializeUserIfNeeded() {
   }
 
   @Override
-  public ResponseEntity<Profile> submitIdVerification(IdVerificationRequest request) {
-    // TODO(dmohs): Prevent this if the user has already attempted verification?
-    Person person = blockscoreService.createPerson(
-      request.getFirstName(), request.getLastName(),
-      new Address()
-        .setStreet1(request.getStreetLine1()).setStreet2(request.getStreetLine2())
-        .setCity(request.getCity()).setSubdivision(request.getState())
-        .setPostalCode(request.getZip()).setCountryCode("US"),
-      request.getDob(),
-      request.getDocumentType(), request.getDocumentNumber()
-    );
+  public ResponseEntity<Profile> submitIdVerification() {
+    User user = userProvider.get();
+    if (user.getRequestedIdVerification() != null) {
```

Comment:
Shouldn't this be == null? Or == null || == false?

You probably don't want to send more than one of these per user.

Reply:
Correct! That was a mistake. Have fixed.